config/audit_log/user_action_events.yml

```diff
@@ -1,4 +1,60 @@
----
 user_login:
```

Comment:
I would update this just to match the structure of the file and for scaling
```
user_login:
  - event_type: authentication
    details: User logged in
```

Reply:
@acaruso-oddball Check out this file: config/appeals_status/mock_get_appeals_responses.yml for list references. This file has some duplicate keys and it organizes those keys by creating lists. And in another file: config/identity_settings/settings.yml, we don't do that because there are no duplicate keys.